package.json

```diff
@@ -28,11 +28,11 @@
     "loader-utils": "^1.1.0"
   },
   "peerDependencies": {
-    "imagemin": "^5.0.0 || ^6.0.0 || ^7.0.0"
+    "imagemin": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0"
```

Comment:
since we are changing the way we get imagemin, we probably are only compatible with `"^8.0.0"`

Reply:
I tried the test with the other versions and it still worked.